src/main/scala/com/aivean/royalroad/Args.scala

```diff
@@ -40,6 +40,11 @@ class Args(args: Seq[String]) extends ScallopConf(args) {
     default = Some("div.chapter-content")
   )
 
+  val keepSiteTitle = opt[Boolean](
+    descr = "Keep the Royal Road site title part of the story in filename",
```

Comment:
Let's not pollute the short flags namespace with rarely-used options.

```suggestion
    descr = "Keep the Royal Road site title part of the story in filename",
    noshort = true,
```

I'd suggest even removing the option altogeter.